abstractions/php/src/Serialization/SerializationWriter.php

```diff
@@ -122,6 +122,20 @@ public function writeDateIntervalValue(?string $key, ?DateInterval $value): void
      */
     public function writeByteValue(?string $key, ?Byte $value): void;
 
+    /**
+     * @param string|null $key
+     * @param array<mixed>|null $value
+     * @return void
+     */
+    public function writeCollectionOfNonParsableObjectValues(?string $key, ?array $value): void;
```

Comment:
what do you think about aligning on Primitive like other languages?
Also, this probably need to be reflected on parse node as well.

Reply:
Done!